docs/source/run/parameters.rst

```diff
@@ -220,8 +226,22 @@ Modeling ion motion is not yet supported by the explicit solver
     Uses a Taylor approximation of the Greens function to solve the Poisson equations with
     open boundary conditions. It's recommended to use this together with
     ``fields.extended_solve = true`` and ``geometry.is_periodic = false false false``.
-    Not implemented for the explicit Helmholtz solver.
+    Only available with the predictor-corrector solver.
+
+Explicit solver parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* ``hipace.use_amrex_mlmg`` (`bool`) optional (default `0`)
+    Whether to use the AMReX multigrid solver. Note that this requires the compile time option ``AMReX_LINEAR_SOLVERS`` to be true. Generally not recommended since it is significantly slower than the default HiPACE++ multigrid solver.
 
+* ``hipace.MG_tolerance_rel`` (`float`) optional (default `1e-4`)
+    Relative error tolerance of the AMReX multigrid solver.
```

Comment:
`MG_tolerance_rel`, `MG_tolerance_abs` and `MG_verbose` affects both MG solvers.

Reply:
Good point, added that